ex1/ex1.c

```diff
@@ -8,7 +8,9 @@
 
 int main(int argc, char *argv[])
 {
-    // Your code here
-
+    int x = 100;
+    printf("your variable is %i\n", x + 2);
+    fork();
```

Comment:
Your program doesn't do anything with the forked child process. Anything that gets printed then is only from the parent process, not from the child process. You need to keep a reference to what `fork` returns and then check that value to see whether you're in the parent context or child context, then have each context alter the variable `x`.